### ISSUE TITLE
Printer not found

### ISSUE THREAD

**Issue by AliaksandrRegino** [star-micronics/StarXpand-SDK-iOS#13]
## Description
Our project uses the StarPRNT iOS SDK (Objective-C), which has worked great for many years. During the project update, we decided to switch to the StarXpand SDK for iOS and ran into a problem of the new framework not working with the TSP143LAN printer.
When searching for available printers, the TSP143LAN printer is not displayed. After searching for printers using StarXpand SDK for iOS, it stops populating in the list of available printers in the application that uses StarPRNT iOS SDK (Objective-C).
After rebooting the printer, it becomes available in the application that uses StarPRNT iOS SDK (Objective-C) until we call startDiscovery() in StarXpand SDK for iOS, after which the printer is no longer available anywhere in the system for both frameworks. This problem only occurs with the TSP143LAN printer. The mC-Print2 and mC-print3 work without any issues.

## Your device where the bug occurs 
 - Device: 
    iPhones and iPads
 - OS:
    iOS 16

## Your printer
- Model Names:
    TSP143LAN
- Interface:
    Ethernet

**Comment by bandit-ibayashi** [comment]
@AliaksandrRegino We are very thankful you have been using our products for a very long time and for migrating to the StarXpand SDK.

We are sorry to say, but TSP143LAN is not supported by this StarXpand SDK (StarIO10), as it has been discontinued for some years. StarIO10 has a process in which the software absorbs the difference in commands depending on the printer model, so that [unsupportedModel](https://star-m.jp/products/s_print/sdk/starxpand/manual/en/ios-swift-api-reference/star-io10-error/index.html#unsupportedmodel-message-string-errorcode-stario10errorcode) is returned if a command is sent for an unsupported model.

As a workaround, we would appreciate it if you continue to use the existing StarPRNT SDK (StarIO library) when using the TSP143LAN, although we are very afraid that you are updating your software.
If you have any questions about support for TSP143LAN, please feel free to contact your regional support team from [this our site](https://starmicronics.com/global/).

Thank you for your understanding.

**Comment by asa47** [comment]
@bandit-ibayashi Hello, I hope you are good! We have a few question regarding the TSP143 models. We understand that all TSP143 printers belongs to the TSP100 family series; but after reading the above answer, there is a bit confusion on our side. Could you please clarify the following questions?

- Is there a difference between TSP143 models and TSP143iii models? 
- Am I correct to assume that StarXpand wont support all the following models TSP143iiiU, TSP143iiiBT and TSP143iiiLAN? 

Thanks!

**Comment by gare-bear** [comment]
@asa47 

> * Is there a difference between TSP143 models and TSP143iii models?

Yes! The TSP143 is an older generation of the same printer (v2). Its been discontinued for some time now and for that reason it is not supported by StarXpand. TSP143III is is the third generation of TSP143.

> * Am I correct to assume that StarXpand wont support all the following models TSP143iiiU, TSP143iiiBT and TSP143iiiLAN?
That is incorrect. StarXpand supports all of these models, and is shown in the [documentation](https://star-m.jp/products/s_print/sdk/starxpand/manual/en/index.html). FYI the docs refer to these printers by their series name (TSP100III).
